csharp/src/Google.Protobuf.Test.CodeGen/Google.Protobuf.Test.CodeGen.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--
+    This CodeGen project is kept seperate from the original test project for many reasons.

```

Comment:
nit: How about Google.Protobuf.Test.GeneratedProtos or just Google.Protobuf.TestProtos
instead of Google.Protobuf.Test.CodeGen as it makes the purposes even clearer?
I don't feel strongly about this, so leaving up to you.

Reply:
Could probably keep it as TestProtos, sure.